tensorboard/webapp/widgets/linked_time_fob/linked_time_types.ts

```diff
@@ -19,3 +19,49 @@ export interface LinkedTime {
   start: {step: number};
   end: {step: number} | null;
 }
+
+/**
+ * This class is intended to be implemented by the card that has a
+ * LinkedTimeFobControllerComponent.
+ *
+ * Each implementer will have some sort of Scale that is used to translate
+ * convert between step and pixel so that the fob lines up with the axis
```

Comment:
one of "translate" or "convert" in "translate convert" is repetitive.

Reply:
done.

Reply:
done.